_data/reference.yaml

```diff
@@ -77,6 +77,13 @@ toc:
     path: /reference/cli/pulumi_stack.html
   - title: Update
     path: /reference/cli/pulumi_update.html
+- title: Cloud Console
```

Comment:
@mehzer Any preference for what we call our "Cloud Console" in the documentation?